test/e2e/e2eutil.go

```diff
@@ -37,14 +40,14 @@ func UpdateMongoDBResource(original *mdbv1.MongoDB, updateFunc func(*mdbv1.Mongo
 // using the provided retryInterval and timeout
 func WaitForConfigMapToExist(cmName string, retryInterval, timeout time.Duration) (corev1.ConfigMap, error) {
 	cm := corev1.ConfigMap{}
-	return cm, waitForRuntimeObjectToExist(cmName, retryInterval, timeout, &cm)
+	return cm, waitForRuntimeObjectToExist(cmName, retryInterval, timeout, &cm, f.Global.OperatorNamespace)
 }
 
 // WaitForSecretToExist waits until a Secret of the given name exists
 // using the provided retryInterval and timeout
-func WaitForSecretToExist(cmName string, retryInterval, timeout time.Duration) (corev1.Secret, error) {
+func WaitForSecretToExist(cmName string, retryInterval, timeout time.Duration, nsp string) (corev1.Secret, error) {
```

Comment:
same as before about `nsp` name, here it's definitely not easy to understand what it means though

Reply:
Done! I changed all instances of nsp in this file to namespace as well for clarity.